src/site/content/en/blog/richer-offline-w-periodic-background-sync/index.md

```diff
@@ -0,0 +1,458 @@
+---
+title: Richer offline experiences with periodic background sync
+subhead:
+  Sync your apps data in the background for a more native-like experience.
+authors:
+  - jeffposnick
+  - joemedley
+date: 2019-11-08
+updated: 2019-11-08
+hero: hero.jpg
+alt: Colorful airplanes flying in sync
+origin_trial:
+  url: https://developers.chrome.com/origintrials/#/view_trial/4048736065006075905
+description: |
+  Periodic Background Sync enables web applications to periodically
+  synchronize data in the background, bringing web apps closer to the behavior
+  of a native app.
+tags:
+  - capabilities
+  - post
+  - progressive-web-apps
+  - service-worker
+---
+
+{% Aside %}
+  The Periodic Background Sync API, part of Google's [capabilities
+  project](https://developers.google.com/web/updates/capabilities), and since
+  Chrome 77 is in an [**origin trial**](#ot).is currently in development. This
+  post will be updated as the implementation progresses.
```

Comment:
```suggestion
  The Periodic Background Sync API, part of Google's [capabilities
  project](https://developers.google.com/web/updates/capabilities), and since
  Chrome 77 in an [**origin trial**](#ot), is currently in development. This
  post will be updated as the implementation progresses.
```